packages/dara-core/tests/js/use-action.spec.tsx

```diff
@@ -938,10 +946,13 @@ describe('useAction', () => {
         const wsClient = new MockWebSocketClient('uid');
 
         const { result } = renderHook(
-            () =>
-                useAction(annotated, {
+            (): [(input?: any) => Promise<void>, boolean] => {
```

Comment:
Wasn't this type changed to just have Promise<void>?

Reply:
Yes, but in this test hook I combine both the useAction and useActionIsLoading hooks into a single return so I can test both at the same time, hence the typing here.

Reply:
Oh oops got the deleted part and added part mixed up thought you were typing useAction, thank you for explaining!